app/helpers/tasks_helper.rb

```diff
@@ -14,6 +14,21 @@ def task_priority_label(task)
                 class: task_priority_label_class(task.priority))
   end
 
+  def has_latitude_longitude?
+    @task.latitude.present? and
+    @task.longitude.present?
+  end
+
+  def task_address
+    city = @task.city
+    city = @task.city + ', ' if @task.city.present?
```

Comment:
Should we add the comma if the `@task.state` is missing ?

Reply:
Good call!